lldb/source/Commands/CommandObjectTarget.cpp

```diff
@@ -4796,6 +4796,17 @@ class CommandObjectTargetStopHookAdd : public CommandObjectParsed,
         m_one_liner.push_back(std::string(option_arg));
         break;
 
+      case 'F': {
+        bool value, success;
+        value = OptionArgParser::ToBoolean(option_arg, false, &success);
+        if (success) {
+          m_at_first_stop = value;
+        } else
+          error = Status::FromErrorStringWithFormat(
+              "invalid boolean value '%s' passed for -F option",
+              option_arg.str().c_str());
```

Comment:
Nit: either have braces for both or neither.
```suggestion
        if (success) {
          m_at_first_stop = value;
        } else {
          error = Status::FromErrorStringWithFormat(
              "invalid boolean value '%s' passed for -F option",
              option_arg.str().c_str());
        }    
```

Reply:
done